flow.js

```diff
@@ -52,11 +52,13 @@ class Flow {
         {condition: 'electricsExtractorCords', nextStep: 'repair-bathroom-electric-problems'},
         {condition: 'sink', nextStep: 'sink-problems'},
         {condition: 'windows', nextStep: 'repair-window-problems'},
+        {condition: 'dampMould', nextStep: 'bathroom-damp-mould-problems'},
```

Comment:
this should be dampOrMould

Reply:
the api expects `dampMould`

Reply:
Theres some inconsistency in the api and frontend. I picked up a dampOrMould ticket and the frontend already had dampOrMould so i used dampOrMould. I guess whatever you pick ensure theres consistency.